ansible/roles/ovn/tasks/start.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: start OVN database
+  docker:
+    name: ovn-database
+    image: "{{ ovn_db_image_full }}"
+    pull: always
+    privileged: yes
+    net: host
+    detach: True
+    state: started
+    command: "ovn_set_database {{ ovn_database_alias_ip}} {{ ovn_database_device }}"
+  when:
+    - inventory_hostname in groups['ovn-database']
+
+- name: start OVN chassis
+  docker:
+    name: "ovn-chassis-{{ item }}"
+    image: "{{ ovn_db_image_full }}"
+    pull: always
+    privileged: yes
+    net: host
+    detach: True
+    state: started
+    command: "ovn_set_chassis --ovn --controller-ip {{ ovn_database_alias_ip}} --host-ip {{ ovn_chassis_start_cidr_ip.split('.')[0] }}.{{ ovn_chassis_start_cidr_ip.split('.')[1] }}.{{ ovn_chassis_start_cidr_ip.split('.')[2] | int + groups['emulation-hosts'].index(inventory_hostname) | int }}.{{ ovn_chassis_start_cidr_ip.split('.')[3] | int + item | int }}/{{ ovn_chassis_cidr_mask }} --device {{ ovn_database_device }}"
```

Comment:
Agree. I will write an ansible python module that can take input of the start_cird and generate a list of cidr for the fake chassis. - Hui